language/evm/hardhat-examples/contracts/Greeter.move

```diff
@@ -1,22 +1,23 @@
 #[contract]
 module Evm::Greeter {
     use Evm::Evm::{self};
+    use Evm::Evm::sign;
 
     #[storage]
     struct State has key {
         greeting: vector<u8>,
     }
 
     // TODO: move-to-yul does not support constructor.
-    // #[create]
-    // public fun create(greeting: vector<u8>) {
-    //     move_to<State>(
-    //         &sign(self()),
-    //         State {
-    //             greeting,
-    //         }
-    //     );
-    // }
+    #[callable(sig=b"create(string)")]
```

Comment:
Don't we need the `#[create]` attribute here to say this function is the constructor though?

Reply:
Yeah, it is just temporary for testing. I will add it back later.